src/components/css/BookItem.module.css

```diff
@@ -1,39 +1,166 @@
-.BookContainer {
-  width: 80%;
-  margin: 0 auto;
-  font-size: 1.75em;
-  border: 1px solid black;
-  display: flex;
-  padding: 1em;
-  justify-content: space-around;
-}
-
-.BookContainer .BookInfo {
-  display: flex;
-  flex-direction: column;
-  gap: 0.5rem;
-}
-
-.BookContainer .BookInfo ul {
-  list-style: none;
-  display: flex;
-  gap: 1em;
-  padding-top: 1em;
-}
-
-.BookContainer .BookInfo ul li a {
-  text-decoration: none;
-}
-
-.Oval {
-  width: 4.25rem;
-  height: 4.25rem;
-  border-style: solid;
-  border-radius: 50%;
-  border-width: 5px;
-  border: 5px solid #307bbe;
-  border-left-color: #e8e8e8;
-  border-image-source: linear-gradient(to bottom, #307bbe, var(--dodger-blue));
-  border-image-slice: 1;
-  transform: rotate(45deg);
-}
+.BookContainer {
+  width: 80%;

```

Comment:
- [ ] The styling has to be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Please add the missing vertical bars ✍🏻 

![18](https://user-images.githubusercontent.com/79058364/191485855-6437c9c2-3925-4785-9f40-09a1e1414f15.png)

Reply:
The vertical bars were not very visible from my end. Let me have another look at the design.

Reply:
Hey @BenMukebo  looking at the design again. I find only one vertical bar exists on it - the one to the right.

Reply:
Hi! just add one bar then!!